public/diagnostic.js

```diff
@@ -23,3 +23,44 @@
 //  -  Order 1 : 2 hats ($5 each)
 //  -  Order 2 : 1 desk lamp ($20 each)
 //  -  Order 3 : 3 hand towels ($8 each)
+
+const Order = Ember.Object.extend({
+  unitPrice: 0,
```

Comment:
These need to be self-referential in Ember, so it knows how to create the getters and setters:
```js
unitPrice: this.unitPrice,
quantity: this.quantity
```